lib/Backends/NNPI/NNPIOptions.h

```diff
@@ -137,7 +140,7 @@ class NNPIBackendOptions : public NNPIOptions {
   DECLARE_NNPI_OPTION(inferOnDevice, bool, "InferOnDevice",
                       "Enable execution on device (if true, will also force "
                       "compilation for HW and ignore the UseIceT option).",
-                      "USE_INF_API", "0");
+                      "USE_INF_API", "1");
```

Comment:
I think we'd prefer to leave the default to ICEREF for now -- any specific motivation for this change?

Reply:
Since our main target is the HW device, we preferred it be the default.

Reply:
Our tools and people are already setup to assume default is 0, so we'd like to leave this as 0 to reduce churn and confusion on our end. Same for the other the other Options defaults for `USE_INF_API` too of course.

Reply:
OK, will change it back to 0 for now.